### DISCUSSION TITLE
is it possible to create a task/command from a language server?

### DISCUSSION THREAD

**Discussion by frederik-uni** [zed-industries/zed#13392] [Feature Requests]
I want to integrate my [live server](https://github.com/frederik-uni/rusty-live-server) into zed. This does more or less exactly the same as the vscode extension. Integrating that into a lsp isn't a problem. I would need a task or a command to open the browser. (task could display log[reload file xyz]). If it is possible, it would be quite interesting how to create a task/command & how the task/command can exchange data with the lsp.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
